frontend/src/features/business/components/VendorDetails.tsx

```diff
@@ -219,7 +219,7 @@ export function VendorDetails({ vendor, nearbyVendors }: VendorDetailsProps) {
                 </Typography>
                 <Typography variant="body1" component="p" sx={{ mb: 2 }}>
                   {vendor.business_name} is a {vendor.specialties.has(VendorSpecialty.SPECIALTY_HAIR) ? 'hair' : ''}{vendor.specialties.size == 2 ? ' and ' : ' '}{vendor.specialties.has(VendorSpecialty.SPECIALTY_MAKEUP) ? 'makeup' : ''} artist
-                  specializing in Asian features. They are based in {resolvedLocation?.toLowerCase()?.includes("area") ? 'the ' : ''} {resolvedLocation}.
+                  specializing in Asian features, based in {resolvedLocation?.toLowerCase()?.includes("area") ? 'the ' : ''} {resolvedLocation}.
```

Comment:
maybe `specializing in Asian features and based in`?

Reply:
updated